src/tests/Microsoft.PowerFx.Core.Tests/TestRunnerTests/File2.txt

```diff
@@ -0,0 +1,11 @@
+﻿
+>> MultiInput
+  secondline
+Result
+
+>> Engines2
+/* Engine1*/ ER1
+/* Engine2*/ ER2
```

Comment:
If we fully embrace this shift, we could get rid of the /*engine*/ support altogether.